resources/local/config.toml

```diff
@@ -244,7 +244,12 @@ max_state_store_size = 10_737_418_240
 # Memory deduplication.
 #
 # If enabled, nodes will attempt to share loaded objects if possible.
-enable_mem_deduplication = false
+enable_mem_deduplication = true
+
+# Memory duplication garbage collection.
+#
+# Sets the frequency how often the memory pool cache is swept for free references.
```

Comment:
Would be good to clarify that "5" means "every 5th time something is put in the pool the cache is swept". (@marc-casperlabs: Please confirm that's correct!)

Reply:
Yes, that's correct! :)